openmdao/components/tests/test_external_code_comp.py

```diff
@@ -273,9 +273,12 @@ def setup(self):
         self.options['external_input_files'] = [self.input_file]
         self.options['external_output_files'] = [self.output_file]
 
-        self.options['command'] = [
-            sys.executable, 'extcode_paraboloid.py', self.input_file, self.output_file
-        ]
+        # If you want to write your command as a list, the code below will also work.
+        # self.options['command'] = [
+        #     sys.executable, 'extcode_paraboloid.py', self.input_file, self.output_file
+        # ]
+
+        self.options['command'] = 'python extcode_paraboloid.py paraboloid_input.dat paraboloid_output.dat'
```

Comment:
I don't think you should hard-code the file names here, since they are specified via attributes.
Should probably do something like:
_cmd_ = `python extcode_paraboloid.py {} {}'.format(self.input_file, self.output_file)`